src/test/components/panels/EventOverviews.test.tsx

```diff
@@ -0,0 +1,217 @@
+import { EventOverviews } from "../../../app/components/elements/panels/EventOverviews";
+import { TestUserRole, renderTestEnvironment } from "../../utils";
+import { screen, within } from "@testing-library/react";
+import {
+  mockUser,
+  mockFutureEventOverviews,
+  mockPastEventOverviews,
+} from "../../../mocks/data";
+import { rest } from "msw";
+import { API_PATH } from "../../../app/services";
+import userEvent from "@testing-library/user-event";
+
+const headers = [
+  "Actions",
+  "Title",
+  "Date",
+  "Booking deadline",
+  "Location",
+  "Status",
+  "Number confirmed",
+  "Number waiting",
+  "Number attended",
+  "Number absent",
+  "Attendance",
+];
+
+const getEventDetails = () => {
```

Comment:
I could be wrong here, but given that you're always returning `rows[1]`, which I guess it means the first event (I would imagine that `rows[0]` is the table header), doesn't this mean that this should be called `getFirstEventDetails`?